README.md

```diff
@@ -135,6 +135,12 @@ For detaching after debugging, you use CTRL+p followed by CTRL+q
 
```

Comment:
Ab Zeile 87 ist noch veralteter Output von `docker-compose ps` drin, wo z.B. rails_test noch auftaucht. Das noch ersetzen. Bei mir sieht das etwa so aus:
```
      Name                     Command               State                  Ports               
------------------------------------------------------------------------------------------------
pbs_cache_1         docker-entrypoint.sh memca ...   Up       11211/tcp                         
pbs_db_1            docker-entrypoint.sh --sor ...   Up       0.0.0.0:33066->3306/tcp, 33060/tcp
pbs_mailcatcher_1   mailcatcher -f --ip 0.0.0.0      Up       0.0.0.0:1080->1080/tcp            
pbs_rails_1         rails-entrypoint rails ser ...   Up       0.0.0.0:3000->3000/tcp            
pbs_sphinx_1        sphinx-start                     Exit 2                                     
pbs_webpack_1       webpack-entrypoint /usr/sr ...   Up       0.0.0.0:3035->3035/tcp            
pbs_worker_1        rails-entrypoint rails job ...   Up                   
```

Wie du siehst ist zudem bei mir der sphinx Container nicht hochgekommen, aber das ist er jeweils auch schon vorher nicht. Hast du dieses Problem nicht? Wenn nicht, können wir das mal separat fixen.

Reply:
sphinx container läuft bei mir. hast du mal das index docker volume gelöscht?

Reply:
Ja, mit `docker-compose down -v` oder? Hat leider nicht geholfen. Kommt weiterhin `FATAL: no valid indexes to serve`. Aber das ist ein Problem für einen anderen Tag.